scripts/release/start.sh

```diff
@@ -59,12 +59,15 @@ echo "Using new version: ${new_version}"
 TMPFILE=$(mktemp "/tmp/DOC_RELEASE.XXXXXX") 
 wget -O "$TMPFILE" https://raw.githubusercontent.com/jaegertracing/documentation/main/RELEASE.md
 
-issue_body=$(python scripts/utils/formatter.py "${TMPFILE}" "${user_version_v1}" "${user_version_v2}")
+# Ensure the UI Release checklist is up to date.
+make init-submodules
```

Comment:
I had a very old version of the jaeger-ui submodule, missing the `BEGIN_CHECKLIST` comment. 
Moreover, this ensures we pick up any changes to the Jaeger UI release process.